integration/jest.config.js

```diff
@@ -9,9 +9,11 @@ module.exports = {
   moduleNameMapper: {
     "^@shapeshiftoss/(.*)": "<rootDir>/../../packages/$1/src",
   },
-  testPathIgnorePatterns: ["dist"],
   globals: {
-    Uint8Array: Uint8Array,
-    ArrayBuffer: ArrayBuffer,
+    'ts-jest': {
+      diagnostics: {
+        ignoreCodes: [7016],
```

Comment:
This is "could not find a declaration file for module" and is caused because of how ts-jest chooses files to include in the compilation process. It will suppress errors during `yarn test`, but that's OK because they'll still occur during `yarn build` if they're valid.

Reply:
Include this explanation as a comment in the file please

Reply:
```suggestion
        // TS(7016) "could not find a declaration file for module" occurs when ts-jest is confused about which *.d.ts to pass
        // as input to TypeScript's compilation API. When this happens, noImplicitAny will complain. Disabling this error will
        // suppress it when run under ts-jest; this is OK, because it will still be reported during the normal build process.
        ignoreCodes: [7016],
```